client/events.lua

```diff
@@ -45,6 +48,7 @@ RegisterNetEvent('QBCore:Command:GoToMarker', function()
             Wait(0)
         end
     end
+    exports['qb-doorlock']:updateDoors()
```

Comment:
To decouple the code from specific resources can you please make this broadcast a client event and then add the update doors event  within qb-doorlock.